src/Credentials/CredentialProvider.php

```diff
@@ -64,7 +65,11 @@ class CredentialProvider
      */
     public static function defaultProvider(array $config = [])
     {
-        $localCredentialProviders = [self::env(), self::ini()];
+        $localCredentialProviders = [
```

Comment:
Should this be its own method just like how the "removeCredentialProviders" method works?